traffic_monitor/experimental/traffic_monitor/manager/datarequest.go

```diff
@@ -606,30 +614,30 @@ func srvAPIVersion(staticAppData StaticAppData) []byte {
 func srvAPITrafficOpsURI(opsConfig OpsConfigThreadsafe) []byte {
 	return []byte(opsConfig.Get().Url)
 }
-func srvAPICacheStates(toData todata.TODataThreadsafe, statHistory StatHistoryThreadsafe, lastHealthDurations DurationMapThreadsafe, localStates peer.CRStatesThreadsafe, lastStats LastStatsThreadsafe, localCacheStatus CacheAvailableStatusThreadsafe) ([]byte, error) {
-	return json.Marshal(createCacheStatuses(toData.Get().ServerTypes, statHistory.Get(), lastHealthDurations.Get(), localStates.Get().Caches, lastStats.Get(), localCacheStatus))
+func srvAPICacheStates(toData todata.TODataThreadsafe, statHistory ResultHistoryThreadsafe, healthHistory ResultHistoryThreadsafe, lastHealthDurations DurationMapThreadsafe, localStates peer.CRStatesThreadsafe, lastStats LastStatsThreadsafe, localCacheStatus CacheAvailableStatusThreadsafe) ([]byte, error) {
+	return json.Marshal(createCacheStatuses(toData.Get().ServerTypes, statHistory.Get(), healthHistory.Get(), lastHealthDurations.Get(), localStates.Get().Caches, lastStats.Get(), localCacheStatus))
 }
 
 func srvAPIBandwidthKbps(toData todata.TODataThreadsafe, lastStats LastStatsThreadsafe) []byte {
-	serverTypes := toData.Get().ServerTypes
+	// serverTypes := toData.Get().ServerTypes
 	kbpsStats := lastStats.Get()
 	sum := float64(0.0)
-	for cache, data := range kbpsStats.Caches {
-		if serverTypes[cache] != enum.CacheTypeEdge {
-			continue
-		}
+	for _, data := range kbpsStats.Caches {
+		// if serverTypes[cache] != enum.CacheTypeEdge {
```

Comment:
I'm not positive TM should be including Mids in bandwidth, that's why I left it. But I'm _almost_ positive TM 1.0 is. Maybe it should be in an `if` statement, with a constant or config?